packages/types/package.json

```
@@ -74,7 +74,9 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/ssz": "^0.14.0",
-    "@lodestar/params": "^1.15.0"
+    "@lodestar/params": "^1.15.0",
+    "@lodestar/utils": "^1.15.0",
+    "ethereum-cryptography": "^1.2.0"
```

we only use v2 of ethereum-cryptography now, see https://github.com/ChainSafe/lodestar/pull/6380

```suggestion
    "ethereum-cryptography": "^2.0.0"
```